lib/reference_counter_api.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/errors/api_error_handling"
+
+# Gets data from reference-counter Toolforge tool
+# https://toolsadmin.wikimedia.org/tools/id/reference-counter
+class ReferenceCounterApi
+  include ApiErrorHandling
+
+  TOOLFORGE_SERVER_URL = 'https://reference-counter.toolforge.org'
+
+  def self.valid_wiki?(wiki)
+    return wiki.project != 'wikidata'
+  end
```

Comment:
As I mentioned in the description, we don't want to use this for wikidata, because that wiki has its own way to calculate references. The new Toolforge API doesn't work for wikidata either.

Reply:
Done on c98338632bb401640eb428fcf446d7a3c54f19e3. But it looks like those commits are not  associated to my github account. Not sure if this is a problem for this specific PR. I guess I did something wrong with `git config`.

Reply:
I think I fixed it. Done on a534520ad1ea38cd26e8479e72afd697113605be